src/core/containers/HandleLogin/index.js

```diff
@@ -33,7 +34,8 @@ class HandleLogin extends React.Component {
   }
 
   render() {
-    const { code, state } = this.props.location.query;
+    const { location } = this.props;
+    const { code, state } = location.query;
     const { error } = this.state;
```

Comment:
If the error state was stored in Redux then we could use that state in Redux tools. For example, if we were styling the pages before and after an error, we could use the time travel slider to toggle back and forth without having to re-execute a login. 

We get all sorts of things like that for free when we use Redux for state management but anytime you manage state yourself, you could get lost.